wagtail/wagtailusers/forms.py

```diff
@@ -17,6 +19,13 @@
     UserPagePermissionsProxy)
 from wagtail.wagtailusers.models import UserProfile
 
+
+DJANGO_GTE_1_9 = django_version >= (1, 9)
```

Comment:
OK, I will change that in a new commit. (Sorry btw, I forgot `[:2]`, which is important since `(1, 10, 'alpha') > (1, 10, 1)` is `True`.)
For the record, I usually do this to avoid doing the attribute lookup, tuple creation and comparison over and over again, and to reduce the risk of typos. It’s not much more complex to find these again after, using PyCharm.